src/Processing.js

```diff
@@ -3256,10 +3256,10 @@
       a2 = ((colorBits2 & PConstants.ALPHA_MASK) >>> 24) / colorModeA;
 
       // Return lerp value for each channel, INT for color, Float for Alpha-range
-      r = p.lerp(r1, r2, amt) | 0;
-      g = p.lerp(g1, g2, amt) | 0;
-      b = p.lerp(b1, b2, amt) | 0;
-      a = p.lerp(a1, a2, amt) * colorModeA;
+      r = p.lerp(r1, r2, amt) + 0.5 | 0;
```

Comment:
same comment as above, we'll need to add some parentheses so |0 applies to the corrected number instead of to 0.5